.github/workflows/process_challenge.yml

```diff
@@ -3,11 +3,11 @@
 
 name: evalai-challenge
 on:
-  push:
```

Comment:
Hey @aditi-dsi , I don't think we can remove this, since we want to run it on every push of the `challenge` branch.